package.json

```diff
@@ -34,12 +40,22 @@
   "devDependencies": {
     "@testing-library/react-hooks": "^2.0.1",
     "axios-mock-adapter": "^1.17.0",
+    "babel-eslint": "^10.0.3",
+    "eslint": "^6.4.0",
+    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-react-hooks": "^2.0.1",
+    "import-sort-cli": "^6.0.0",
+    "import-sort-parser-babylon": "^6.0.0",
+    "import-sort-style-module": "^6.0.0",
     "jest": "^24.8.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^1.18.2",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-test-renderer": "^16.9.0"
   },
   "dependencies": {
-    "axios": "^0.19.0"
+    "axios": "^0.19.0",
+    "lodash.isequal": "^4.5.0"
```

Comment:
seems a shame to bring any lodash in :/

Mapbox elected to use their own function: https://github.com/mapbox/mapbox-gl-js/issues/5744

Worth considering? Genuine open question / mind.

Reply:
I'm bringing in just a single function though, not the whole package?

Reply:
so was mapbox. definitely prefer this to the whole lib and it's certinaly not a blocker, i just wondered if it was worth discussing